### DISCUSSION TITLE
Borderless Window?

### DISCUSSION THREAD

**Discussion by Sarund9** [picoe/Eto#1926] [Q&A] [answered]
I'd like to know how to Make a cross-platform borderless window
Such as the windows of VS code and VS studio

**Comment by InfinityGhost** [comment]
[Window.WindowStyle](http://pages.picoe.ca/docs/api/html/P_Eto_Forms_Window_WindowStyle.htm) sets the style of window borders, which in your case you would want `WindowStyle.None`.

**Comment by Sarund9** [comment, marked as answer]
Doing that results in this:
![Borderless Window](https://user-images.githubusercontent.com/53350334/114910169-f0f2f480-9df3-11eb-8a24-15b0584237ce.png)
Which is not the best result, is there a way to completely remove the border?

**Reply by InfinityGhost** [reply]
That might actually be an inconsistency, on GTK there is no border as the documentation states. 
![image](https://user-images.githubusercontent.com/39218853/114911426-fc8eed00-9dec-11eb-8499-7afb6c124866.png)

**Reply by Sarund9** [reply]
Oh, i was in WPF, not GTK
GTK is giving me missing library errors
(DllNotFoundException: Gtk: libgtk-3-0.dll, libgtk-3.so.0, libgtk-3.0.dylib, gtk-3.dll)
How do I install these?

**Reply by InfinityGhost** [reply]
> Oh, i was in WPF, not GTK
> GTK is giving me missing library errors
> (DllNotFoundException: Gtk: libgtk-3-0.dll, libgtk-3.so.0, libgtk-3.0.dylib, gtk-3.dll)
> How do I install these?

I'm not familiar with how to use GTK on Windows, I'm running Linux. What I'm saying is that this might need to be propagated to an issue being that there is an inconsistency between WPF and GTK.
